doc/wiki/source_code/Version_Control_History.md

```diff
@@ -0,0 +1,109 @@
+# Version Control History
```

Comment:
Is this file new file or renaming of an old file? Where is the corresponding change of link/reference to this file?

Reply:
It is a new file. I will update links at the end after I have finished editing the file itself

Reply:
It looks like most of its content is coming from `doc/wiki/source_code/Version_Control_System.md`, which is removed in the second commit. If the content is moved from one file to another file, both the removal and addition need be in the same commit so it shows how the content is moved. Better yet, there should be one commit just moving the content without changes, and a second commit showing the changes.